Assets/Scripts/MIDI2Event/MIDI2EventUnity.cs

```diff
@@ -68,4 +75,22 @@ public Action Subscribe(
     {
         return eventPlayer.Subscribe(action, note, octave, type);
     }
+
+    //returns the current number of seconds per beat
+    public float SecPerBeat()
+    {
+        return (float)eventPlayer.SecPerBeat();
+    }
+
+    //returns the current number of seconds per beat
+    public float BeatPerSec()
+    {
+        return (float)eventPlayer.BeatPerSec();
+    }
+
+    //returns whether the system is currently playing
+    public bool IsPlaying()
+    {
+        return eventPlayer.IsPlaying();
+    }
```

Comment:
Again, as explained in the Midi2Event class comment, these could potentially be Properties, but it's your choice

Reply:
changed!